dockerfile/integrator/Dockerfile

```diff
@@ -14,16 +13,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-
+#
 # ------------------------------------------------------------------------
```

Comment:
This license header seems to be slightly different from the rest of the license headers.

Reply:
We have kept the same style on all the recent product-wide refinements made to dockerfiles. Thus this change.